packages/encryption/src/hashRipemd160.ts

```diff
@@ -47,17 +46,17 @@ export class NodeCryptoRipemd160Digest implements Ripemd160Digest {
 }
 
 export function createHashRipemd160() {
-  const nodeCryptoCreateHash = isNodeCryptoAvailable(nodeCrypto => {
-    if (typeof nodeCrypto.createHash === 'function') {
-      return nodeCrypto.createHash;
-    }
-    return false;
-  });
-  if (nodeCryptoCreateHash) {
-    return new NodeCryptoRipemd160Digest(nodeCryptoCreateHash);
-  } else {
-    return new Ripemd160PolyfillDigest();
-  }
+  // // disable node hashRipemd160, because it doesn't work with node 17,18 openSSL
```

Comment:
_Commented out the node version of this hashing function._ Maybe we should switch to js version everywhere. Would simplify code a bunch. The only thing against it would be potential loss of performance, where node might switch to native openSSL.